de-lib/src/main/java/org/iplantc/de/theme/base/client/admin/BelphegorDisplayStrings.properties

```diff
@@ -17,7 +17,7 @@ addCategoryPrompt = New Category
 appCategorizeSuccess = Successfully added {0} to the following Categories: {1,list}.
 appCategorizeSuccess[\=1] = Successfully added {0} to the following Category: {1,list}.
 confirmDeleteAppCategory = Do you want to delete the category \"{0}\"?
-confirmDeleteAppTitle = Delete App
+confirmDeprecateAppTitle = Delete App
```

Comment:
That's a good point. Since you put it that way, it does make sense to rename the properties and method names for the display strings to match the term used in the strings.

For non-display-string-methods and fields (like in the Views), I would still stick with `delete` over `deprecate`, especially since the term `delete` is still used in a lot of places.